data.sql

```diff
@@ -1,6 +1,16 @@
 /* Populate database with sample data. */
 
+/* Task One */
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235749642-b590e9cf-9303-4c13-ac25-7010f91bb86d.png)

**[OPTIONAL]**

- [ ] It would be nice if you fix these links in your Table of Contents which you can see in the image above to make your readme descriptive and professional 😊